doc/lua_api.txt

```diff
@@ -5320,6 +5320,13 @@ Call these functions only at load time!
     * `pos_list` is an array of all modified positions.
     * `node_list` is an array of the old node that was previously at the position
       with the corresponding index in pos_list.
+* `minetest.register_on_mapblocks_changed(function(modified_blocks, modified_block_count))`
+    * Called soon after any nodes or node metadata have been modified. Will never
+      be called more than once per server step. No map edit events are skipped.
```

Comment:
- the mapblock positions are obviously of the ones that were modified, but this is never spelled out
- no modder knows what a "map edit event" is, not sure why this is being mentioned
- possibility of false-positives not mentioned
  (may trigger despite no observable changes from Lua)